### PR TITLE
[Session Replication] LPD-32691 Add functional test to test session replication

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:environment

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
ci:test:clustering

**Comment by julschong** [comment]
ci:stop

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
ci:test:clustering

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:clustering

**Comment by julschong** [comment]
ci:test:environments

File: modules/apps/portal/portal-cluster-multiple-sample-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -33,42 +34,60 @@ ClusterSampleData clusterSampleData = new ClusterSampleData();
...
 </ul>
 
+<div class="logged-in-session-count">
+	<h4> Logged In Session Count: </h4>
+
+	<%
+	int count = 0;
+
+	for (HttpSession httpSession : PortalSessionContext.values()) {
+		if (httpSession.getAttribute("USER_ID") != null) {
```
Comment (julschong) [inline modules/apps/portal/portal-cluster-multiple-sample-web/src/main/resources/META-INF/resources/view.jsp:44]: inlined httpSession.getAttribute("USER_ID")

File: modules/apps/portal/portal-cluster-multiple-sample-web/src/main/resources/META-INF/resources/view.jsp [hunk 2]
```diff
@@ -33,42 +34,60 @@ ClusterSampleData clusterSampleData = new ClusterSampleData();
 	</li>
 </ul>
 
+<div class="logged-in-session-count">
+	<h4> Logged In Session Count: </h4>
+
+	<%
+	int count = 0;
```
Comment (julschong) [inline modules/apps/portal/portal-cluster-multiple-sample-web/src/main/resources/META-INF/resources/view.jsp:41]: fixed variable name

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
ci:test:clustering

**Comment by julschong** [comment]
@tinatian per our discussion this covers first 3 actions:
- Remove CanSessionReplicateBetweenNodes , this old test case never ran on environment and logic can be covered by new functional tests.
- Merge in new functional tests without environment.acceptance = true first.
- Leave old testCanModuleBeShutdownDuringReplicateSession as "ignored" for now

I'm running clustering test to make sure this is good. Will be ready after test report is good

**Comment by julschong** [comment]
ci:report:52147843

**Comment by julschong** [comment]
@tinatian both new test cases passed, ready for review

**Comment by julschong** [comment]
changed commit message for [7ec7486](https://github.com/liferay-core-infra/liferay-portal/pull/7304/commits/7ec7486d58225ee3bd8d2e31b4153a4006c5da2d) code is the same

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/153233

**Pull request closed without merging**